mobile/src/Student/screens/StudentGameIntro/IntroInfo/index.js

```diff
@@ -24,16 +30,54 @@ const styles = StyleSheet.create({
         alignContent: 'center',
         backgroundColor: 'rgba(0,0,0,0)',
     },
-    image: {
-        // width: win.width,
-        // height: win.width,
+    phone: {
+        //width: win.width,
+        //height: win.width,
         resizeMode: 'cover',
     },
+    phaseSticker: {
+        marginRight: scale(-55),
+        marginTop: verticalScale(80),
+        zIndex: 1
+    },
+    imgContainer: {
+        flexDirection: 'row'
+    },
     text: {
         textAlign: 'center',
         textAlignVertical: 'bottom',
         fontFamily: fontFamilies.montserratBold,
-        fontSize: fonts.xxMedium,
+        fontSize: 20,
```

Comment:
Lets add this to `theme.js` and use `fonts.` to fetch it. This helps us to have all font sizes in one place and add dynamic text sizing for accessibility later. You can also use the closes one in there instead of adding a new one